samples/notebookinstance_lifecycle_config/README.md

```diff
@@ -0,0 +1,38 @@
+# Notebook Instance Lifecycle Configuration sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Create a Notebook Instance Lifecycle Configuration
+This command creates a Notebook Instance Lifecycle configuration.
+Note: The onCreate and onStart fields must be in base64.
+
+In the example crd the value of the onCreate field decodes to: `echo "Entering onCreate"` and the value of the onStart field decodes to: `echo "Entering onStart"`
```

Comment:
not sure what you mean by `example crd`

Reply:
IMO it is better to have these conversions listed as a comment in the yaml itself instead of here - what do others think ?

Reply:
I agree that it's better in the yaml file.